www/md-plugins/rehype-auto-toc.js

```diff
@@ -0,0 +1,83 @@
+export default function transform() {
```

Comment:
This is just a fork of a fork of a custom markdown plugin I have lying around. The Starlight one is too deeply-tied to Starlight internals. I don’t know of another common one that’s easy-to-use. But when you look at how they’re made, they’re actually really really simple, as you can see here it’s < 100 lines, and all it’s doing is just crawling the markdown AST once and making a nested `<ol>` out of it.